### DISCUSSION TITLE
Relay support

### DISCUSSION THREAD

**Discussion by l-you** [thecodingmachine/graphqlite#501] [Ideas]
Any idea on implementing [Relay server specification](https://relay.dev/docs/guides/graphql-server-specification/)?

**Comment by oojacoboo** [comment]
I don't really see what's so special about this.  You can easily create a query that takes in a UUID and return an instance of a "dumb" interface - today, with virtually any GQL server implementation.  This mostly defeats the entire purpose of GraphQL though, as it's a bastardization of the strong typing that makes GraphQL so great.

Am I missing something here?
